modules/apps/27-interchain-accounts/controller/client/cli/tx.go

```diff
@@ -29,8 +32,8 @@ func newRegisterInterchainAccountCmd() *cobra.Command {
 		Long: strings.TrimSpace(`Register an account on the counterparty chain via the 
 connection id from the source chain. Connection identifier should be for the source chain 
 and the interchain account will be created on the counterparty chain. Callers are expected to 
-provide the appropriate application version string via {version} flag. Generates a new 
-port identifier using the provided owner string, binds to the port identifier and claims 
+provide the appropriate application version string via {version} flag and the desired ordering
+via the {ordering} flag. Generates a new port identifier using the provided owner string, binds to the port identifier and claims 
```

Comment:
Maybe we could add an example or some extra documentation to specify that the ordering string needs to be `order_ordered` or `order_unordered`, so that users know what string to use and what options are available.

Reply:
yup, agreed! Added `supportedOrderings` in the flag `usage`